src/Esfa.Vacancy.Manage.Api/Controllers/CreateApprenticeshipController.cs

```diff
@@ -173,6 +173,8 @@ public CreateApprenticeshipController(CreateApprenticeshipOrchestrator orchestra
         /// | 31041       | Invalid Supplementary question 2            |
         /// | 31042       | Invalid External application url            |
         /// | 31043       | Invalid External Application Instructions   |
+        /// | 31044       | Invalid Is Employer Disability Confident    |
+        /// | 31045       | Invalid Is Additional Location Information  |
         ///                                                                                                  
         /// </summary>
```

Comment:
Do we need to add any comments in the help text for Is Employer Disability Confident eg acceptable values etc?